### ISSUE TITLE
ffplay crashing during capture

### ISSUE THREAD

**Issue by lawrenceaj** [amiaopensource/vrecord#485]
Hi-

ffplay keeps crashing and closing the vrecord window at around the 1 hour mark, which is causing our transfers to end prematurely. I've attached the logs, but I'm not seeing any obvious errors. I just ran `brew update ` and restarted our computer to no avail. Has anyone else encountered this issue? 

Best,

AJ 

[SC_0001_20200109_Montgomery_VHSC_01_ffv1_capture_options.log](https://github.com/amiaopensource/vrecord/files/4042054/SC_0001_20200109_Montgomery_VHSC_01_ffv1_capture_options.log)
[SC_0001_20200109_Montgomery_VHSC_01_ffv1_ffmpeg_decklink_input.log](https://github.com/amiaopensource/vrecord/files/4042055/SC_0001_20200109_Montgomery_VHSC_01_ffv1_ffmpeg_decklink_input.log)

**Comment by libbyhopfauf** [comment]
@lawrenceaj  What filter are you using for capturing? On the MIPoPS setup, we have noticed that unless we use "unfiltered" for capturing that we encounter crashing and buffer overruns consistently. Also, are you using the GUI or the command line? We have to close all the way out of the GUI after passthrough mode to successfully have vrecord switch over to "unfiltered" for capturing (vs. "visual+numerical" which we use for passthrough generally).

**Comment by lawrenceaj** [comment]
Yes, we've been using Visual + Numerical and the command line for capture. I'll try using Unfiltered and report back.

**Comment by dericed** [comment]
Hey @lawrenceaj, nudge on those tests, particularly with the new release.

**Comment by lawrenceaj** [comment]
We encountered the same problem with ffplay when using Unfiltered, but I'll upgrade to the new release and see where things are.

**Comment by dericed** [comment]
Hey @lawrenceaj, a warning that the files you referenced were done with release v2020-01-27, which had a bug which in some cases, the audio would be in vorbis. @privatezero fixed it a few days later. Vrecord should have given an error via mediaconch in your case, since the file didn't match the expectations.

**Comment by privatezero** [comment]
Closing  - hopefully this has been addressed in subsequent releases. Let us know if the problem still persists!
